code/blocks/AccordionBlock.php

```diff
@@ -55,4 +55,48 @@ public function getCMSFields()
 
         return $fields;
     }
+
+    /**
+     * @return DataList
+     */
+    public function getPanelList()
+    {
+        if ($this->Panels()->exists()) {
+            return $this->Panels()->sort('Sort');
+        }
+        return $this->Panels();
+    }
+}
+
+class AccordionBlock_Controller extends Block_Controller
+{
+    /**
+     * @var string
+     */
+    private $accodion_class = 'accordion-block';
```

Comment:
I think we need the `static` declared as well for this to be used by the `AccordionClass()` method. This is also declared in the `yml` file, but that should be deferred to the php option I think: `private static $accordion_class = 'accordion-block';` here.